HW_2/task_3.py

```diff
@@ -0,0 +1,29 @@
+"""
+Задайте список размерностью N. Каждый элемент списка вычисляется выражением (1 + 1 / n) ** n,
+где n – позиция (не индекс) элемента в списке, причем 1 < n < N.
+Выведите сумму элементов списка. Ответ округлите до сотых.
+
+Ввод: значение типа <int>
+Вывод: значение типа <float>
+
+Пример:
+1
+2.0
+
+2
+4.25
+
+3
+6.62
+"""
+
+n = int(input("Введите число n: "))
+my_list = round(sum([(1 + 1 / i) ** i for i in range(1, n+1)]), 2)
+print(my_list)
+
+exit()
```

Comment:
Не самый хороший способ завершить работу программы. Но если вам действительно необходимо это сделать таким образом, то лучше делать так:
```suggestion
sys.exit()
```
Ну и соответствующую библиотеку, конечно же, импортировать.

Reply:
подскажите как импортировать библиотеку?